src/guide/advanced/triggers.md

```diff
@@ -124,6 +128,9 @@ the detailed information on how to use each of them:
 - By-call triggers
 - Block-based triggers
```

Comment:
I think this section needs updating. In Iroha 2 we have:

- Data triggers
- Time triggers
  - Scheduled triggers
  - Pre-commit triggers
- By-call triggers

So

1. As you write below -- all triggers are *Event* triggers. The difference is in the event type. I think by `Event triggers` here you mean `Data triggers`, which are triggered when something happenned with some data like domain, account and etc.
2. `pre-commit triggers` is one sub-type of `time triggers`.
3. There aren't any `block-based triggers`. I think you mean `pre-commit triggers` here -- they are executed just before every new block commit.